dd-sdk-android/src/main/kotlin/com/datadog/android/sessionreplay/internal/exception/InvalidPayloadFormatException.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.exception
+
+internal class InvalidPayloadFormatException(message: String) : Exception(message)
```

Comment:
Let's make it extend `RuntimeException`, so that it is not checked one.

```suggestion
internal class InvalidPayloadFormatException(message: String) : RuntimeException(message)
```